hub/package-manager/winget/install.md

```diff
@@ -43,6 +43,7 @@ The options allow you to customize the install experience to meet your needs.
 | **-v, --version**  |  Enables you to specify an exact version to install. If not specified, latest will install the highest versioned application. |  
 | **-s, --source**   |  Restricts the search to the source name provided. Must be followed by the source name. |  
 | **--scope**   |  Allows you to specify if the installer should target user or machine scope. |  
+| **-a, --architecture**   |  Select the architecture to install. | 
```

Comment:
Should we provide an example?
"Select the architecture to install. For example, --architecture x86."

Reply:
Will defer to @denelon as he may have a stronger opinion. 

The changes I made are exactly aligned with what I see in the client help display.  I am hesitant to add an example here as that would make it seem slightly out of place. Is there a better place we can show examples?

Reply:
I think we have some examples, and we should include one for arm64 as an example.

Reply:
I decided to not give an example here so that it is consistent with the other argument descriptions that also don't have examples. I think just a clear description here would be sufficient.